javaslang/src/test/java/javaslang/collection/AbstractSetTest.java

```diff
@@ -14,6 +14,8 @@
     @Override
     abstract protected <T> Set<T> empty();
 
+    abstract protected <T> Set<T> emptyWithNull();
```

Comment:
\[Checkstyle\] ERROR: 'protected' modifier out of order with the JLS suggestions\.